module/vision/BallDetector/src/BallDetector.cpp

```diff
@@ -233,62 +225,57 @@ namespace module::vision {
                         // Point on line = camera = Hcw.topRightCorner<3, 1>()
                         Eigen::Affine3f Hcw(horizon.Hcw.cast<float>());
                         const float d = (Hcw.inverse().translation().z() - field.ball_radius) / std::abs(axis.z());
-                        const Eigen::Vector3f rBCc      = axis * d;
-                        const float projection_distance = rBCc.norm();
+                        const Eigen::Vector3f srBCc     = axis * d;
+                        const float projection_distance = srBCc.norm();
```

Comment:
doesn't `d == srBCc.norm()` by definition? (this assumes `axis` is a unit length, which seems reasonable, because it's "normalized" above)

Reply:
Afaik... `d` is only the distance (height difference to the center of the ball from the camera?), `axis` is the observation plane vector to the center of the cluster of the ball. `srBCc` is the spherical vector from Camera to Ball center in camera space. But honestly idfk  @Bidski @TrentHouliston